codegen/tests/simple/app.go

```diff
@@ -0,0 +1,34 @@
+// Code generated by sysl DO NOT EDIT.
+package simple
+
+import (
+	"github.com/anz-bank/sysl-go/codegen/tests/deps"
+	"github.com/anz-bank/sysl-go/config"
+	"github.com/anz-bank/sysl-go/core"
+	"github.com/anz-bank/sysl-go/handlerinitialiser"
+)
+
+// App for Simple
+type App struct {
+}
+
+// HandlerManager for Simple
+type HandlerManager struct {
+	coreCfg             *config.DefaultConfig
+	enabledHandlers     []handlerinitialiser.HandlerInitialiser
+	enabledGrpcHandlers []handlerinitialiser.GrpcHandlerInitialiser
+}
+
+// LoadServices ...
+func (a *App) InitialiseHandler(h *HandlerManager, serviceInterface ServiceInterface, serviceCallback core.RestGenCallback) error {
+	depsHTTPClient, depsErr := core.BuildDownstreamHTTPClient("deps", &h.coreCfg.GenCode.Downstream.(*DownstreamConfig).Deps)
```

Comment:
"deps" should be the name of the system (or some other meaningful identifier).

Reply:
deps is the app name picked up from the model.